test/fixtures/eslint-comments/no-use.fail.ts

```diff
@@ -1 +1,11 @@
-/* eslint no-undef: off / / eslint-env browser / / eslint-disable foo / / eslint-enable bar / // eslint-disable-line // eslint-disable-next-line / exported foo / / global $ / / globals a, b, c */
```

Comment:
I don't think this worked correctly because of how it was weirdly single-line. Looks like originally this was also copied from the [docs](https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/no-use.html#rule-details), so I did the same, only keeping the comment styles as is